docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  public-api-dev:
+    image: itprotv/stack:tag-2.1.3
+    volumes:
+      - stack-root:/stack-root      
+      - .:$PROJECT_DIR
+      - stack-work:/$PROJECT_DIR/.stack-work
```

Comment:
Do we need to make a separate volume for `stack-work` when we already have a volume for the current directory?

Reply:
Probably not! I'll remove it!